addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1871,6 +1871,77 @@ QUnit.test('chat window does not fetch messages if hidden', async function (asse
     );
 });
 
+QUnit.test('chat window of a chat new message separator behaviour on receiving new message [REQUIRE FOCUS]', async function (assert) {
```

Comment:
Name of test is too broad. It should be more explicit to what it is asserted. Splitting into 2 tests would help naming the tests correctly.